docs_zh-CN/tutorials/config.md

```diff
@@ -1 +1,377 @@
 # 教程 1: 学习配置文件
+
+我们整合了模块和继承设计到我们的配置里，这便于做很多实验。如果您像检查配置文件，您可以运行 `python tools/print_config.py /PATH/TO/CONFIG` 去查看完整的配置文件。您还可以传递参数
+`--options xxx.yyy=zzz` 去查看更新的配置。
+
+## 配置文件的结构
+
+在 `config/_base_` 下面有4个基本构成成分(component)： 数据集(dataset)，模型(model)，计划表(schedule)和默认运行时间(default runtime)。
+这样，像 DeepLabV3, PSPNet 这样的模型可以容易地被构造。被来自 `_base_` 的构成成分所组成的配置文件叫做 `_primitive_`。
+
+对于所有在同一个文件夹下的配置文件，推荐**只有一个** primitive 配置文件。所有其他的配置文件都应该继承自这个 primitive 配置文件。以这样的方式，最大的继承深度是3。
```

Comment:
We may refer to https://mmaction2.readthedocs.io/zh_CN/latest/tutorials/1_config.html.

Reply:
> We may refer to https://mmaction2.readthedocs.io/zh_CN/latest/tutorials/1_config.html.

OK, I will follow and check that doc as template.